extensions/cornerstone/src/initMeasurementService.js

```diff
@@ -252,7 +253,24 @@ const connectMeasurementServiceToTools = (
         SOPInstanceUID
       );
 
-      const imageId = dataSource.getImageIdsForInstance({ instance });
+      let imageId;
+      let frameNumber = 1;
+
+      if (measurement.metadata && measurement.metadata.referencedImageId) {
```

Comment:
this can be simiplified to `measurement?.metadata?.referencedImageId`

Reply:
fixed in a9d22efe494178b6cbbdaaf971700f4b9e1bf517